index.html

```diff
@@ -4,16 +4,26 @@
     <meta charset="UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
-    <title>Title</title>
-    <link rel="stylesheet" href="styles/App.scss"/>
+    <title>Movie</title>
+    <link rel="stylesheet" href="styles/App.css"/>
 
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
           rel="stylesheet">
 </head>
 <body>
+    <div id="header">
```

Comment:
Generally we should be using semantic html with HTML5 standars, so that should be the <Header/ > tag